diff_drive_controller/include/diff_drive_controller/diff_drive_controller.h

```diff
@@ -171,6 +171,9 @@ namespace diff_drive_controller{
     /// Whether to publish odometry to tf or not:
     bool enable_odom_tf_;
 
+    /// whether to define the back as the front of the robot instead
+    bool swap_front_and_back_;
```

Comment:
IMO reverse forward direction makes more sense but it does not matter

Reply:
I can change it, I also don't particularly like that name. How about `invert_forward_direction`?